packages/e2e/cypress/integration/common/shared.ts

```diff
@@ -159,7 +159,9 @@ Then('I see tab {string}', (search: string) => {
 });
 
 Then('I see {string}', (message: string) => {
-  cy.findByRole('document').contains(new RegExp(escapeRegExp(message), 'i'));
+  cy.findByRole('document')
+    .contains(new RegExp(escapeRegExp(message), 'i'))
+    .should('be.visible');
```

Comment:
Small bug we had: `contains` does not fail if it does not find any result. So I'm chaining it with `be.visible` here.